Cargo.lock

```diff
@@ -1,33 +1,122 @@
 # This file is automatically @generated by Cargo.
 # It is not intended for manual editing.
-version = 3
```

Comment:
This cargo lock was generated on a machine using `cargo 1.84.0-nightly (cf53cc54b 2024-10-18)`. It looks like in the intervening years since this package was last updated, Cargo.lock's format has been updated to include a lot more information